src/components/BusRouteDetails.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import busIcon from "../assets/bus-icon.png";
+import ghostIcon from "../assets/bxs_ghost.png";
+
+import ridershipData from "../Routes/july2022_cta_ridership_data_day_type_summary.json";
+
+function BusRouteDetails({ selectedRoute, busFraction, percentileKeys }) {
+  const selectedRouteRidership = ridershipData.filter(
+    (x) => x.route_id === selectedRoute[0].properties.route_id
+  );
+
+  const averageRidershipPerWeekday = selectedRouteRidership.find(
+    (x) => x.day_type === "weekday"
+  );
+
+  //percentile
+  const percentileIndex = Number(
+    (selectedRoute[0].properties.percentiles * 100).toLocaleString("en-US", {
+      minimumIntegerDigits: 2,
+      useGrouping: false,
+    })[0]
+  );
+
+  const barGraphBars = percentileKeys.map((x, index) => {
+    return (
+      <div
+        key={x}
+        className={`bargraph-bar ${
+          index === percentileIndex ? "selected-bar" : ""
+        }`}
+        style={{ height: x }}
+      ></div>
+    );
+  });
+
+  return (
+    <div className="bus-route-details">
+      <h3>
+        <span className="bus-number">
+          {selectedRoute[0].properties.route_id}
```

Comment:
What do you think about having a variable to refer to `selectedRoute[0]`, since we are always referring to the first selected route. Also, the name would also reflect the idea that it's _the_ route that has been selected.